_data-prepper/pipelines/configuration/sinks/s3.md

```diff
@@ -76,11 +76,11 @@ Use the following options when customizing the `s3` sink.
 
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
-`bucket` | Yes | String | The name of the S3 bucket to which the sink writes.
-`codec` | Yes | [Codec](#codec) | The codec that determines how the data is serialized in the S3 object.
+`bucket` | Yes | String | The object from which the data is retrieved and then stored. The `name` must match the name of your object store.
+`codec` | Yes | [Codec](#codec) | The format of the output data.
```

Comment:
I like this change overall. But, I also think it should clearly connect to codecs since it has to be an existing Data Prepper codec.

> The codec determining the format of output data